build/CoreGraphics/lib/CoreGraphicsLib.vcxproj

```diff
@@ -55,6 +55,10 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGPDFStream.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGPDFString.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGShading.mm" />
+    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\D2DWrapper.mm" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\include\CoreGraphics\D2DWrapper.h" />
```

Comment:
shouldn't it be in Frameworks\CoreGraphics if it's a private header, then?

---
In reply to: [82040170](https://github.com/Microsoft/WinObjC/pull/1085#discussion_r82040170) [](ancestors = 82040170,82025651)